File: src/main/java/org/jukeboxmc/potion/DarknessEffect.java
package org.jukeboxmc.potion;

import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class DarknessEffect extends Effect {

    @Override
    public int getId() {
        return 30;
    }

    @Override
    public @NotNull EffectType getEffectType() {
        return EffectType.DARKNESS;
    }

    @Override
    public @NotNull Color getEffectColor() {
        return new Color( 41, 39, 33 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/DarknessEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class DarknessEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 30
    }

    override fun getEffectType(): EffectType {
        return EffectType.DARKNESS
    }

    override fun getEffectColor(): Color {
        return Color(41, 39, 33)
    }

    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {

    }
}